pkg/storage/stores/indexshipper/shipper.go

```diff
@@ -70,6 +70,10 @@ type Config struct {
 	IngesterDBRetainPeriod time.Duration
 }
 
+func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
+	cfg.RegisterFlagsWithPrefix("", f)
+}
+
```

Comment:
This change is unrelated, right? Could be reverted.

Reply:
It was causing the new test I added to fail while setting the defaults, and it should have been added in the first place anyways.